### PR TITLE
Added travis-ci build support

### PR DESCRIPTION
**Pull request by erseco**
Added support for travis-ci.org, you should activate it to check if the code is compiling correctly in every PR or commit.

Also I've added a badge in the top of the README.md file, after activating the https://travis-ci.org/exofoundation/CivX-Qt/ you must edit that file to point to your build

### CODE REVIEW

**Comment by wxmerkt** [comment]
Awesome, can we squash this PR to keep things clean if it gets merged?

**Comment by wxmerkt** [comment]
Does this actually build? There is no `Makefile` in the root of the repository and the `build.sh` uses:

```bash
cd src
sudo make -j$NPROC -f makefile.unix
sudo cp civxd /usr/bin/civxd
```

^ which isn't good practice (don't use `sudo` with make, usually use install routines instead of direct copy - and local/bin is preferred over system bin)

**Comment by erseco** [comment]
I'm building just calling `qmake` and `make`, I'm avoiding the original `build.sh` because of sudo as you say.

**Comment by erseco** [comment]
Any news about this PR?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/erseco/civxd.svg?branch=master)](https://travis-ci.org/erseco/civxd)
```
Comment (wxmerkt) [inline README.md]: This would need to be updated prior to merging

Comment (erseco) [reply README.md]: Solved!

**Pull request merged**